zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/test/java/org/apache/zookeeper/metrics/prometheus/PrometheusMetricsProviderTest.java

```diff
@@ -57,7 +62,7 @@ public void setup() throws Exception {
         provider = new PrometheusMetricsProvider();
         Properties configuration = new Properties();
         configuration.setProperty("httpHost", "127.0.0.1"); // local host for test
-        configuration.setProperty("httpPort", "0"); // ephemeral port
+        configuration.setProperty("httpPort", String.valueOf(METRICS_PORT));
```

Comment:
it is better to have an ephemeral port, 
otherwise it is easy to break the execution of the test on an env in which port 7000 is already bound and also it prevents running concurrent tests

Reply:
I have modified the test so that it uses an ephemeral port.